diff --git a/node_modules/electron-notarize/lib/index.js b/node_modules/electron-notarize/lib/index.js
index 801e595..db5999a 100644
--- a/node_modules/electron-notarize/lib/index.js
+++ b/node_modules/electron-notarize/lib/index.js
@@ -160,7 +160,7 @@ function stapleApp(opts) {
                     return [4 /*yield*/, spawn_1.spawn('xcrun', [
                             'stapler',
                             'staple',
-                            '-v',
+                            '-vvvv',
                             path.basename(opts.appPath),
                         ], {
                             cwd: path.dirname(opts.appPath),
@@ -170,6 +170,7 @@ function stapleApp(opts) {
                     if (result.code !== 0) {
                         throw new Error("Failed to staple your application with code: " + result.code + "\n\n" + result.output);
                     }
+                    d('staple finished' + "\n\n" + result.output)
                     d('staple succeeded');
                     return [2 /*return*/];
             }
