diff --git a/node_modules/electron-packager/node_modules/electron-notarize/lib/index.js b/node_modules/electron-packager/node_modules/electron-notarize/lib/index.js
index f47fdf1..d7ee8ec 100644
--- a/node_modules/electron-packager/node_modules/electron-notarize/lib/index.js
+++ b/node_modules/electron-packager/node_modules/electron-notarize/lib/index.js
@@ -130,6 +130,7 @@ function stapleApp(opts) {
         if (result.code !== 0) {
             throw new Error(`Failed to staple your application with code: ${result.code}\n\n${result.output}`);
         }
+        d('staple finished' + "\n\n" + result.output)
         d('staple succeeded');
         return;
     });
